runtime/lua/vim/_editor.lua

```diff
@@ -1176,9 +1176,97 @@ function vim._init_default_autocmds()
   })
 end
 
+--- Parse a string of hex chars as a color.
+---
+--- The string can contain 1 to 4 hex characters. The returned value is a
+--- fractional value between 0 and 1 representing the intensity of the color in
+--- the range.
+---
+--- For instance, if only a single hex char "a" is used, then this function
+--- returns 0.625 (10 / 16), while a value of "aa" would return 0.664 (170 /
+--- 256).
+---
+--- @param c string Color as a string of hex chars
+--- @return number Intensity of the color
+local function hexcolor(c)
```

Comment:
```suggestion
local function luminance(c)
```
to match the description below? (should use the same term in all docstrings and variable names)